hsmd/hsmd.c

```diff
@@ -2133,6 +2149,10 @@ int main(int argc, char *argv[])
 	status_setup_async(status_conn);
 	uintmap_init(&clients);
 
+	/* We start off uninitialized, and require the first message
+	 * to be WIRE_HSMD_INIT. */
+	initialized = false;
```

Comment:
Generally prefer an explicit initialization at declaration, so it's not reliant on what order things are called in.

(In this case it's implicitly initialized to false, but I still prefer explicit).

Reply:
Wan't sure which rule prevails here (don't init until you need, or the init early), but it makes sense here, since we rely on it being set from the very beginning.

PR updated.